src/indexer/allo/v2/handleEvent.ts

```diff
@@ -204,12 +230,18 @@ export async function handleEvent(
             address: strategyAddress,
           });
           break;
+        case "allov2.DirectGrantsSimpleStrategy":
+          subscribeToContract({
+            contract: "AlloV2/DirectGrantsSimpleStrategy/V1",
+            address: strategyAddress,
+          });
+          break;
       }
 
-      let applicationsStartTime = new Date();
-      let applicationsEndTime = new Date();
-      let donationsStartTime = new Date();
-      let donationsEndTime = new Date();
+      let applicationsStartTime: Date | null = new Date();
```

Comment:
it be less error prone if the initial values were null, not sure if there's ever a time that we want to set these dates to the current dates